material/src/main/java/org/odk/collect/material/ErrorsPill.kt

```diff
@@ -25,9 +25,6 @@ class ErrorsPill(context: Context, attrs: AttributeSet?) : MaterialPill(context,
             visibility = View.VISIBLE
             setIcon(org.odk.collect.icons.R.drawable.ic_baseline_check_24)
             setText(org.odk.collect.strings.R.string.draft_no_errors)
-            setPillBackgroundColor(MaterialColors.getColor(this, com.google.android.material.R.attr.colorSurfaceContainerHighest))
```

Comment:
This will be still needed because when used in a recycler view those pills might be reused and we need to make sure the colors are updated. You can create multiple drafts with and without errors and scroll fast to see that they will have wrong colors.